Update roassal

badetitou opened the pull request:
It also remove some piece of buggy piece of code (because of the Pharo vm.... ie Esteban)

jecisc commented on src/Famix-Tests/FamixTWithInheritancesTest.class.st:
```diff
@@ -14,3 +14,20 @@ self entity: (MooseEntity new copyWithTalent:FamixTWithInheritances).
 ]
+
+{ #category : #tests }
+FamixTWithInheritancesTest >> testSuperInheritances [
```
The test comes from a trait I think and here you compiled it in the class using the trait I guess.

badetitou committed:
testNewHash
skip on CI
use isClassSide
bah nan c'etait bien
formating
remove automatic transformation
rewrite transformWith
double ``
rewrite
flag it
flag 2
flag in comment
remove all
move to trait method
Revert "move to trait method"

This reverts commit 70d1a90674fb1aaad7274ecc37906b68282fdd05.
